qt/python/mantidqt/mantidqt/widgets/sliceviewer/models/model.py

```diff
@@ -576,13 +576,16 @@ def _cut_names(self, cut: str):
 
         return xcut_name, ycut_name, help_msg
 
-    def get_hkl_from_xyz(self, xdim, ydim, zdim, xdata, ydata, zdata):
+    def get_hkl_from_full_point(self, full_point: List[float], qdims_i: List[int]):
```

Comment:
As discussed offline, the `qdims_i` are actually a property that can be determined from the workspace and so I think ultimately belongs in the model (not retrieved from the data view dimension widget and passed via the presenter) - but I'm happy for that to be addressed in #35095